### PR TITLE
Add log for debugging

### PR DESCRIPTION
**Pull request by leepengxiang**

### CODE REVIEW

File: exporter/exporterwrapper/exporterwrapper.go [hunk 1]
```diff
@@ -88,6 +96,10 @@ func PushOcProtoSpansToOCTraceExporter(ocExporter trace.Exporter, td data.TraceD
 			errs = append(errs, err)
 		}
 	}
+	log.Printf("spans: %d\tgoogle spans: %d\n", len(td.Spans), len(goodSpans))
+	log.Printf("https://pantheon.corp.google.com/traces/traces?project=cloud-debugging&tid=%x%x\n",
+		binary.LittleEndian.Uint64(goodSpans[0].TraceId[0:8]),
```
Comment (bogdandrutu) [inline exporter/exporterwrapper/exporterwrapper.go]: They do BigEndian.

File: exporter/exporterwrapper/exporterwrapper.go [hunk 2]
```diff
@@ -80,6 +82,12 @@ func PushOcProtoSpansToOCTraceExporter(ocExporter trace.Exporter, td data.TraceD
 	var errs []error
 	var goodSpans []*tracepb.Span
 	for _, span := range td.Spans {
+		log.Printf("Trace ID: %x%x\tSpan ID: %x\tParent Span ID: %x\tName: %s\n",
+			binary.LittleEndian.Uint64(span.TraceId[0:8]),
+			binary.LittleEndian.Uint64(span.TraceId[8:16]),
+			binary.LittleEndian.Uint64(span.SpanId),
+			binary.LittleEndian.Uint64(span.ParentSpanId),
```
Comment (bogdandrutu) [inline exporter/exporterwrapper/exporterwrapper.go]: Print nil here probably if ParentSpanId is nil.

File: exporter/exporterwrapper/exporterwrapper.go (hunk 1 above)
Comment (leepengxiang) [reply exporter/exporterwrapper/exporterwrapper.go]: Done.

File: exporter/exporterwrapper/exporterwrapper.go (hunk 2 above)
Comment (leepengxiang) [reply exporter/exporterwrapper/exporterwrapper.go]: Done.

**Pull request merged**